src/05-excitation-transfer.md

```diff
@@ -0,0 +1,762 @@
+P, psi = simulate(H, psi0, times)
+```
+
+We need to recreate the bra-ket labels because we are now only working with even parity states.
+
+```python
+bra_labels, ket_labels = make_braket_labels(nmm_list)
+```
+
+Now let's plot the results using a helper function `plot_prob` that pulls together plotting code that we used in the last tutorial.
+
+```python
+plot_prob(P, times, ket_labels)
+plt.title(f"2 TSS with {H_latex}  ($\Delta E \\approx 1.9328$, $\omega=1$, $U=0.1$)   (Fig 8)");
+```
+
+Fig 8 shows exactly what we expected i.e. both TSS transition from "+" to "-" each giving of 2 bosons in the process, i.e. $|0,+, + \rangle \rightarrow |4,-, - \rangle$.
+
+
+### 5.6.2 - Mystery crossings
+
+
+In Fig 7, we found something unexpected for the energy levels corresponding to no overall excitation, i.e. those consisting of equal number of "+" and "-". We found that e.g. levels 2 and 3 appeared to cross each other. Considering that we are in an even parity universe, the expectation is that all levels can interact with each other and create anti-crossings in the process - so why don't levels 2 and 3 do this?
+
+To dig a little deeper, we'll set $\Delta E = 0$ and inspect levels 2 and 3 in more detail - specifically we'll look at their corresponding eigenstates.
+
+```python
+DeltaE = 0
+```
+
+```python
+H = DeltaE*two_state_1 + DeltaE*two_state_2 + 1*bosons + 0.1*interaction_1 + 0.1*interaction_2
+```
+
+```python
+evals, ekets = H.eigenstates()
+```
+
+Let's first look at the energy of levels 2 and 3
+
+```python
+print(f"Energy of level 2 = {evals[2]}")
+print(f"Energy of level 3 = {evals[3]}")
+```
+
+How intriguing, the energy of level 3 is exactly 1.5 - just as as it was when we didn't have any coupling between the TSS and the boson field.
+
+Now let's look at the corresponding eigenstates using a helper function `prettify_states` that we imported from the helper file.
+
+```python
+prettify_states(ekets, nmm_list)[[2,3]]
+```
+
+We can see that the eigenstates corresponding to levels 2 and 3 are normalised versions of: 
+- Level 2 = $|1,+,-\rangle  + |1,-,+\rangle$ +  other states with boson number different from 1
+- Level 3 = $|1,+,-\rangle  - |1,-,+\rangle$ 
+
+This pattern persists for all $\Delta E$ - try it out for yourself by changing the $\Delta E$ in the above Hamiltonian. There is always a horizontal level corresponding to  $|1,+,-\rangle  - |1,-,+\rangle$ with energy 1.5. This level (and others like it with different boson number) does not couple to others and so does indeed live in its own sub-universe.
+
+Why is it that changing plus to minus in $|1,+,-\rangle  \pm |1,-,+\rangle$ makes such a difference? What is the nature of this other sub-universe? They are important questions but they take us somewhat away from the primary mission of this tutorial. We'll come back to this in the next tutorial.
+
+> For those who just can't wait that long, it has to do with conservation of spin angular momentum
```

Comment:
I would say: conservation of total pseudo-spin 
OR total pseudo-angular momentum